src/models/position/facade.js

```diff
@@ -183,11 +183,11 @@ const prepareChangeParticipantPositionTransaction = async (transferList) => {
           const { transfer, transferState, rawMessage, transferAmount } = reservedTransfers[transferId]
           if (new MLNumber(availablePositionBasedOnLiquidityCover).toNumber() < transferAmount.toNumber()) {
             transferState.transferStateId = Enum.Transfers.TransferInternalState.ABORTED_REJECTED
-            transferState.reason = ErrorHandler.Enums.FSPIOPErrorCodes.PAYER_FSP_INSUFFICIENT_LIQUIDITY.message
+            transferState.reason = 'PAYER_FSP_INSUFFICIENT_LIQUIDITY'
```

Comment:
Please add missing FSPIOPErrorCodes enums to this library --> https://github.com/mojaloop/central-services-error-handling/blob/master/src/enums.js#L78.

Same comment applies to the reason below.

Reply:
I refactored the fix with a different approach